### ISSUE TITLE
Different styling while dragging.

### ISSUE THREAD

**Issue by johman10** [bevacqua/dragula#61]
Hi there!

Thank you so much for Dragula! It's pretty awesome and works as promised out of the box. I just have one little issue. While dragging the width of the object isn't kept the same. Before dragging it looks like this:
![schermafbeelding 2015-05-12 om 16 34 00](https://cloud.githubusercontent.com/assets/5371454/7588324/1f843046-f8c5-11e4-86d3-4586c079041a.png)
While dragging it looks like this:
![schermafbeelding 2015-05-12 om 16 34 40](https://cloud.githubusercontent.com/assets/5371454/7588336/283f9216-f8c5-11e4-9190-614140705295.png)
And after dropping it looks just as it should be.

Do you have any idea what the problem could be? If you need a codepen I can probably fix that! ;)
Thanks in advance!

**Comment by bevacqua** [comment]
CodePen would be nice. But basically: when dragging, your stuff is removed from its container, and appended to `document.body`. So any styles your element inherited from its parent, or any styles affected by siblings of your element, are gone now. Take that into account when adjusting the styles for your drag shadow.

**Comment by johman10** [comment]
Made a codepen but it doesn't seem to do the same.
In the codepen it hides the internal objects or something like that.
See for yourself over here:
[http://codepen.io/johman10/pen/XbdNoo](http://codepen.io/johman10/pen/XbdNoo)

Hope it shows the problem.
I'm thinking it has to do with scss nesting... could that be it?

**Comment by bevacqua** [comment]
All your styles inherit from `.options_menu`. The `options_menu` class is nowhere in the dragged element.

**Comment by johman10** [comment]
Wow! That was pretty stupid! Should have seen that myself!

Thanks for you help and thanks for Dragula! Gotta love it!

**Comment by bevacqua** [comment]
Also fixed the example in the demo page for reference

**Comment by bevacqua** [comment]
https://github.com/bevacqua/dragula/blob/4d2c87bc14295bdb1c50dc9f18e1b9de0d135c58/example/example.css#L116-L130

**Comment by aderchox** [comment]
Hi, a somewhat related question.
**How do I change the cursor from `grab` to `grabbing` while dragging?**
I've tried adding `cursor: grabbing` to both `.gu-mirror` and `.gu-unselectable` (which are the ones that seem to be related), but neither one is working.
This is in Svelte:
```css
  .drgwrapper :is(:global(.gu-unselectable), :global(.gu-mirror)) {
    cursor: grabbing;
  }
```
I've also changed them inside `/node_modules/dragula/dist/dragula.css` (and restarted bundler), still without success.
I'm using Chrome.
